src/app/components/LiveHeaderMedia/index.test.tsx

```diff
@@ -233,6 +166,73 @@ describe('liveMediaStream', () => {
     expect(window.mediaPlayers.p0gh4n67.play).toHaveBeenCalledTimes(playCalls);
   });
 
+  describe('Visually hidden text', () => {
```

Comment:
Can we try to more closely replicate the tests which have been deleted? Let's not focus too much on the visually hidden text on its own, but rather what the entire text content is. 

I'd like to see in these 2 tests:

1. Open State: Title with no punctuation, Contains some upsetting scenes.Watch
2. Close State: Close video, Title with punctuation! Contains some upsetting scenes.